zhaquirks/tuya/ts0601_trv_sas.py

```diff
@@ -81,6 +85,20 @@ def _update_attribute(self, attrid, value):
             self.endpoint.device.thermostat_bus.listener_event(
                 LOCAL_TEMP_REPORTED, value
             )
+        elif attrid == SCHEDULE_ENABLED_COMMAND_ID:
+            self.endpoint.device.thermostat_bus.listener_event(
+                SCHEDULE_STATE_REPORTED, value
+            )
+
+    async def write_attributes(self, attributes, manufacturer=None):
+        if "system_mode" in attributes or self.attributes_by_name["system_mode"].id in attributes:
```

Comment:
Could someone explain to me why `attributes` passed to that method has numerical `command_id` instead of the command name as keys?

I expected:

```python
{"system_mode": 1}
```

but it receives:

```python
{357: 1}
```

Reply:
Ok, I've figured it out. Its overwritten in `TuyaThermostatCluster:write_attributes()` with value from `ThermostatCluster:map_attribute()`.

https://github.com/zigpy/zha-device-handlers/blob/2292c4edfd5dc0ca6aac4439923c11cefbe5006c/zhaquirks/tuya/__init__.py#L643